README.md

```diff
@@ -1,2 +1,2 @@
 # account-validator-api
-A backend service allowing users to test XBRL validation on their accounts.
+Used to create, read, update, delete company exemptions resources.
```

Comment:
This doesn't seem correct to me. This service has nothing to do with company exemptions. Is this leftover from the project you based this skeleton from?

Reply:
yeah left over I will delete it